models/bamf_nnunet_mr_breast/config/default.yml

```diff
@@ -0,0 +1,67 @@
+general:
+  data_base_dir: /app/data
+  version: 1.0
+  description: Default configuration for Bamf NNUnet Breast and tumor segmentation on MR scans (dicom to dicom)
+
+segdb:
+  triplets:
+    T_BREAST_FGT_STRUCTURE:
+      code: C114467
+      meaning: Breast Fibroglandular Tissue
+      scheme_designator: NCIt
+    T_BREAST_CARCINOMA_MASS:
+      code: 4147007
+      meaning: Mass
+      scheme_designator: SCT
+  segments:
+    FGT:
+      name: FGT
+      category: C_RADIOLOGIC_FINDING
+      type: T_BREAST_FGT_STRUCTURE
+      color: [128, 174, 128]
+    BREAST_CARCINOMA:
+      name: Breast_Carcinoma
+      category: C_MORPHOLOGICALLY_ABNORMAL_STRUCTURE
+      type: T_BREAST_CARCINOMA_MASS
+      modifier: M_RIGHT_AND_LEFT
+      color: [144, 238, 144]
+
+execute:
+- DicomImporter
+- NiftiConverter
+- module: NNUnetRunnerV2
+  nnunet_dataset: Dataset009_Breast
+  roi: BREAST,FGT
```

Comment:
I think it should be `BREAST+FGT` (although FGT is always breast-specific - what do you think?).

Reply:
If changed, changes also need to be applied in the meta.json output section.